modules/runner/src/integrationTest/sql/subquery/exists/test_correlated_exists.test_ignore

```diff
@@ -13,9 +13,29 @@ PRAGMA enable_verification
 statement ok
 CREATE TABLE integers(i INTEGER)
 
+statement ok
+CREATE TABLE t1(a INTEGER, b INTEGER, c INTEGER, d INTEGER, e INTEGER, f INTEGER)
+
+statement ok
+INSERT INTO t1 VALUES (1, 2, 6, 4, 5, 6), (2, 3, 6, 5, 6, 7)
+
 statement ok
 INSERT INTO integers VALUES (1), (2), (3), (NULL)
 
+query IIIIII
```

Comment:
we already have this very test in `select1.test` script

Reply:
in test1 returned resultset not checked, here i fill appropriate data and check result too.

Reply:
then why do we have such a test at all?